src/routes/footer/contact-us.js

```diff
@@ -0,0 +1,10 @@
+export const contactUs = {
+  method: 'GET',
+  path: '/contact-help',
+  handler: (_request, h) => {
+    return h.view('footer/contact-help', {
+      pageTitle: 'Contact us for help',
```

Comment:
The design spec from prototype states that the title tag should be
`Contact us for help - Manage your land and farm businesses - GOV.UK`

Title tag is currently being rendered as this:
```
<title>  Contact us for help | Single Front Door
</title>
```

Reply:
Discussed with Amirs and UCD - this is still WIP therefore acceptable.